wizeline/wize-q-front
# WIP-226 - Docker integration files

#### What does this PR do?
Adds the necessary files for docker image creation
#### Where should the reviewer start?
check the env.sample file and check that you have the same env vars in your .env file. Then make sure the project runs as usual
#### How should this be manually tested?
1. Install docker
2. Login to docker with the wizeline credentials: `docker login devops.wize.mx:5000`
3. Retrieve the docker image `docker pull devops.wize.mx:5000/wl-node-6:2017-03-23`
4. build the image `docker build -t wizeq-front .`
5. Run the container: `docker run --rm -ti  -p 3000:3000 --env-file /path/to/your/project/.env --name wize-front-instance wizeq-front bash start.sh`
6. It should be up and running! Try entering [http://localhost:3000](http://localhost:3000) in your browser.  The login screen should be displayed

#### Any background context you want to provide?
Configuration files based on [https://github.com/wizeline/docker-base-images/tree/master/node-6](https://github.com/wizeline/docker-base-images/tree/master/node-6) 
#### What are the relevant tickets?
WIP-226
